operator/pkg/controllers/master/kubeapiserver.go

```diff
@@ -466,6 +470,15 @@ func apiServerPodSpecForVersion(version string, defaultSpec *v1.PodSpec) v1.PodS
 			args = append(args, arg)
 		}
 		defaultSpec.Containers[0].Args = args
+	case "1.26", "1.27":
+		args := []string{}
```

Comment:
Saw the following logic repeat multiple times. It should be encapsulated in a function.